mesonbuild/scripts/coverage.py

```diff
@@ -218,8 +194,12 @@ def run(args: T.List[str]) -> int:
                         const='sonarqube', help='generate Sonarqube Xml report')
     parser.add_argument('--html', dest='outputs', action='append_const',
                         const='html', help='generate Html report')
-    parser.add_argument('--use_llvm_cov', action='store_true',
+    parser.add_argument('--use-llvm-cov', action='store_true',
                         help='use llvm-cov')
+    parser.add_argument('--gcovr', action='store', default='',
+                        help='The gcovr executable to use if specified')
+    parser.add_argument('--llvm-cov', action='store', default='',
+                        help='The llvm-cov executable to use if specified')
```

Comment:
These options should be added to scripts/coverage.py in the same commit where ninjabackend starts generating rules that require the option to exist.

Otherwise, `git bisect` breaks on the previous commit.